extensions/wikia/AdEngine/js/SevenOneMediaHelper.js

```diff
@@ -39,7 +39,7 @@ define('ext.wikia.adEngine.sevenOneMediaHelper', [
 			},
 			'fullbanner2': {
 				SOI_FB2: true,
-				SOI_PB: true,    // powerbanner (728x180)
+				SOI_PB: false,   // powerbanner (728x180)
```

Comment:
Yeah, the one from spreadsheet with the URL: http://de.gta.wikia.com/wiki/GTA_Wiki?showroom=miwoli&subsite=71i_showroom&yshowroom=pb is red. So, it's `SOI_PB`... it's quite confusing `PB`, `PD`, `BB`, `WP`, `FP`...